atlas-search/public-api/index.md

```diff
@@ -0,0 +1,101 @@
+# Index documentation
+
+### Overview
+
+The index API includes two mutations for creating or updating a document, and deleting a document.
+
+- The `index` mutation accepts a `DocumentInput` and returns a `DocumentMutationResponse`.
+- The `delete` mutation accepts an `id` as a parameter and also returns a `DocumentMutationResponse`.
+
+### Authentication
+
+Clients calling the index API are required to add an authentication header with the valid authentication token.
+
+```
+Authorization: Token {ACCESS_TOKEN}
+```
+
+### Creating or Updating a Document
+
+To create or update a document in the index using the mutation, you need to:
+
+1. Use the `index` operation.
+2. Provide a `DocumentInput` object as a parameter.
+3. In the `DocumentInput` object, provide the necessary document data as a `JSON` object.
+4. Provide a unique identifier for the document in the `id` field of the `DocumentInput` object. If no id is provided, one will be generated for the document.
+
+Here’s an example mutation to create a new document:
+
+```graphql
```

Comment:
would be good to add an example to create a document with provided id

Reply:
Changed the text a little bit. Because providing an id can either create the document if there is no document with same id in the server or update it if it already exists